includes/admin_import.php

```diff
@@ -151,48 +157,130 @@ function tsml_import_page()
+                            $message .= "</tbody></table>";
+
+                            //reset the data source meetings count for this feed
+                            $tsml_data_sources[$data_source_url]['count_meetings'] = tsml_count_meetings();
+
+                            //save data source configuration
+                            update_option('tsml_data_sources', $tsml_data_sources);
+
+                        } else {
+                            $message = __('<p>Your meeting list is already in sync with the records in the ' . $data_source_name . ' feed.<p>', '12-step-meeting-list');
+                        }
+                        tsml_alert($message, 'success');
+                        $stop_processing = true;
+
+                    } else {
+                        $message = __('<p>Your meeting list is being updated to sync with those in the ' . $data_source_name .' feed.</p>', '12-step-meeting-list');
+                        $message .= "<table border='1'><tbody><tr><th>$tbl_col1_txt</th><th>$tbl_col2_txt</th><th>$tbl_col3_txt</th><th>$tbl_col4_txt</th><th>$tbl_col5_txt</th></tr>";
+                        $message .= implode('', $message_lines);
+                        $message .= "</tbody></table>";
+                        tsml_alert($message, 'success');
+                    }
+                }
+
+                if (!$stop_processing) {
+
+                    $tsml_data_sources[$data_source_url] = [
+                                    'status' => 'OK',
+                                    'last_import' => current_time('timestamp'),
+                                    'count_meetings' => 0,
+                                    'name' => $data_source_name,
+                                    'parent_region_id' => $data_source_parent_region_id,
+                                    'type' => 'JSON',
+                                ]; 
+
+                    if ($is_new_parent_feed_import) {  
+
+                        // Create a cron job to run at least once daily 
+                        tsml_schedule_import_scan($data_source_url, $data_source_name);
+                        
+                        //import All the new feed records
+						tsml_import_buffer_set($meetings, $data_source_url, $data_source_parent_region_id);
+
+                    } elseif ($is_feed_refresh) {
+
+	                    //import ONLY the change detected import feed records
+                        tsml_import_buffer_set($import_updates, $data_source_url, $data_source_parent_region_id);
+
+
+                    } else { //load everything
+
+                        //import All the existing feed records
+                        tsml_import_buffer_set($meetings, $data_source_url, $data_source_parent_region_id);
+                    }
+
+                    //reset the data source meetings count for this feed
+                    $tsml_data_sources[$data_source_url]['count_meetings'] = tsml_count_meetings();
```

Comment:
^ This seems to returning full count of all meetings, total, rather than just the meetings for this individual feed. Not sure if that's intended, structure would imply you just want to catch the meetings from the source.